options_compiled.js

```diff
@@ -5975,10 +5975,7 @@ function saveOptions() {
                     status.textContent = '';
                 }, 750);
             });
-        document.getElementById('optionsTextArea').value = beautify(optionsStr, {
-            indent_size: 4,
-            space_in_empty_paren: true
-        });
+
```

Comment:
Why are we getting rid of "beautify" here?

Reply:
We shouldn't, that was a mistake - woops!